scripts/fast-reboot

```diff
@@ -185,6 +185,9 @@ case "$REBOOT_TYPE" in
             # source mlnx-ffb.sh file with
             # functions to check ISSU upgrade possibility
             source mlnx-ffb.sh
+        elif [[ "$sonic_asic_type" == "barefoot" ]]; then
+            REBOOT_TYPE="fastfast-reboot"
+            BOOT_TYPE_ARG="fastfast"
```

Comment:
can you combine this with 182 line, then only on mellanox platform do the source mlnx-ffb.sh.

Reply:
for my understanding, logically expression 
```
if [  barefoot || mellanox ]; then
....
   if [ mellanox ]; then
....
```
looks a bit more complicated than existing one. Besides, we already have the same expression in `warm-reboot` script so, it would be better to keep code identical in a similar handling context.